.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Support Question
+    url: https://discourse.laminas.dev/c/questions/5
```

Comment:
Maybe we should have here link:
https://discourse.laminas.dev/c/questions/expressive/7

And for laminas we have:
https://discourse.laminas.dev/c/questions/components/6

Reply:
Good point.

This brings a new task: update the forum. (I will ask @weierophinney if I can do that.)

Reply:
I agree. But at this time I'm not sure those categories will remain. They might be just renamed or might be archived.

Reply:
I hope we can rename them and old links will be redirected, but not sure if it is possible?

Reply:
@michalbundyra  Category text name does not matter, it is just for link readability. It is routed by id.

Reply:
We have now:

* https://discourse.laminas.dev/c/questions/components/6
* https://discourse.laminas.dev/c/questions/mezzio/7
* https://discourse.laminas.dev/c/questions/api-tools/8

(See also: https://meta.discourse.org/t/how-to-redirect-a-category-using-permalink-rules/43380)

Reply:
I agree that we should link to the specific category for the given project/subproject. @froschdesign has already renamed them (as he's noted in the comment prior to mine). I think with that change, we can get these merged against all three orgs.